### ISSUE TITLE
exa reports unknown argument -p

### ISSUE THREAD

**Issue by zhuzhzh** [ajeetdsouza/zoxide#424]
I use 'exa' instead of `ls`. then `zi` will report the following errors in cshell.

> exa:Unknown argument -p

```
 41     alias ls 'exa -G  --color auto  -s type'
 42     alias ll 'exa -l --color always  -s type'
 43     alias la 'exa -l --color always -a -s type'
 44     alias lsi 'exa -G  --color auto  -s type --icons'
 45     alias lli 'exa -l --color always  -s type --icons'
 46     alias lai 'exa -l --color always -a -s type --icons'
```

I checked the code. If I didn't set `FZF_DEFAULT_OPTS`,  zoxide will pass '-p' to ls.

```
command.args(&[r"--preview=\command -p ls -p {2..}", "--preview-window=down"]);
```
Can we use some options to define the argument passed to ls?

Regards,
Harris

**Comment by ajeetdsouza** [comment]
Hi Harris, what version of zoxide are you on? This should ideally have been fixed on zoxide v0.8.1.

**Comment by zhuzhzh** [comment]
I am still using 0.8.0.  I will upgrade my zoxide version. THX!
